rss.py

```diff
@@ -17,27 +18,32 @@ def setup(self, *args, **kwargs):
 
         for feed in self.feeds:
             if feed not in self.storage:
-                self.storage[feed] = ''
+                self.storage[feed] = []
         self.loop_task(self.update_interval, self.check_feeds, now=False)
 
+    def hash(self, entry):
+        return hashlib.sha224("{}{}{}".format(entry.id,
```

Comment:
@pepol sha224 just looks cool =)

You are right on the ID, not sure how I missed that but in my tests this seemed to behave in a pretty stable manner.

In any case, I personally think we are only interested in the title and the link anyway so I am ditching the ID.